src/api/video/video.controller.js

```diff
@@ -35,5 +36,14 @@ async function getVideos(req, res, next) {
     return next(error);
   }
 }
+const fetchVideos = async (req, res, next) => {
+  const maxResults = req.query.maxResults || 10;
+  try {
+    const response = await fetchLatestYoutubeVideos(maxResults);
+    return res.json(response);
+  } catch (err) {
+    return next(new Error(err));
```

Comment:
`err` would already be an Error instance so there is no need to construct a new error. 
```js
next(err);
```
is good.